### ISSUE TITLE
Running LaTeX.js in Windows gives error

### ISSUE THREAD

**Issue by BinaryAura** [michael-brade/LaTeX.js#94]
After installing npm and latex.js as the README.md shows running latex.js yields the following error:

![latex js_issue](https://user-images.githubusercontent.com/5960287/69560416-529d8f00-0f71-11ea-8285-8fe9270fafc5.PNG)

**Comment by michael-brade** [comment]
Hm... I guess this might mean that PowerShell cannot execute the standard shell scripts, because line 1 is
```
#!/usr/bin/env node
```
So the  '#' is a problem... I don't know Windows/PowerShell, so no idea what to do here.

**Comment by callum-atwal** [comment]
Run it as `node latex.js` ?

**Comment by michael-brade** [comment]
yeah... that idea would work, but you would have to add the full path to `latex.js` - which is kind of ugly. Can't we install a binary on Windows that works when typing just `latex.js`?
